components/Stands/StandCard.tsx

```diff
@@ -0,0 +1,305 @@
+      {CompanyCardContent}
+    </StandardContainer>
+  );
+};
+
+const Live: React.FC<LiveProps> = ({ active }) => (
+  <LiveContainer active={active}>LIVE</LiveContainer>
+);
+
+const Divider = styled.hr`
+  width: 100%;
+  border: none;
+  height: 1.5px;
+  background-color: #f1f1f1;
+  flex-shrink: 0;
+`;
+
+const EventTitle = styled.span``;
+
+const StandardContainer = styled(NudgeDiv)`
+  display: flex;
+  flex-direction: column;
+  max-width: 100vw;
+  height: 180px;
+  background: #ffffff;
+  box-shadow: 0px 0px 4px rgba(0, 0, 0, 0.25);
+  border-radius: 7px;
+  padding: 15px;
+`;
+
+const SPContainer = styled(StandardContainer)`
+  display: flex;
+  flex-direction: column;
+  max-width: 100vw;
+  flex: 1 1 300px;
+  height: 180px;
+  margin: var(--gap) 0 0 var(--gap);
+  background: #ffffff;
+  box-shadow: 0px 0px 4px rgba(0, 0, 0, 0.25);
+  border-radius: 7px;
+  padding: 15px;
+`;
+
+const HSPContainer = styled(SPContainer)`
+  margin-bottom: 25px;
+  grid-column: -1/1;
+  display: grid;
+  grid-template-columns: 40% 60%;
+  @media only screen and (max-width: 1199px) {
+    display: flex;
+  }  ;
+`;
+
+const CompanyEvents = styled.div`
+  align-self: center;
+  padding: 0 20px 0 20px;
+`;
+
+const FirstRow = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+`;
+
+const CompanyImgContainer = styled.div`
+  display: flex;
+  justify-content: flex-start;
+  height: 60px;
+  width: 70%;
+`;
+
+const HSPCompanyImgContainer = styled.div`
+  display: flex;
+  padding: 20px;
+  border-right: 1.5px solid #f1f1f1;
+`;
+
+const FlexContainer = styled.div`
+  display: flex;
+`;
+
+const EventGrid = styled.div`
+  display: grid;
+  grid-template-columns: 20% 80%;
+  padding-bottom: 8px;
+`;
+
+const CompanyImg = styled.img`
+  object-fit: contain;
+  width: auto;
+  height: 100%;
+`;
+
+const HSPCompanyImg = styled(CompanyImg)``;
+
+const CompanyInfo = styled.div`
+  display: flex;
+  height: 100%;
+  flex-direction: column;
+  justify-content: space-between;
+`;
+
+const CurrentEvent = styled.div`
+  display: flex;
+  flex-direction: column;
+  color: #555;
+`;
+
+const SubHeader = styled.span`
+  font-weight: 600;
+`;
+
+const TimeSlot = styled.span<{ current?: boolean }>`
+  font-weight: ${(props) => (props.current ? 600 : 200)};
+`;
+
+const LiveContainer = styled.div<{ active: boolean }>`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-color: #ffffff;
+  flex: 0 0 50px;
+  height: 25px;
+  padding: 4px;
+  border-radius: 3px;
+  text-align: center;
+  stroke: 2px;
+  border: 1px solid ${(props) => (props.active ? 'red' : 'grey')};
+  color: ${(props) => (props.active ? 'red' : 'grey')};
+  text-decoration: ${(props) => (props.active ? 'none' : 'line-through')};
+`;
+
+export default createFragmentContainer(StandCard, {
+  company: graphql`
+    fragment StandCard_company on Company {
+      name
+      id
+      description
```

Comment:
Remove unused properties from the query, there are some that I can't see are used anywhere.